### ISSUE TITLE
conda package cant find requirements.txt

### ISSUE THREAD

**Issue by dhirschfeld** [skorch-dev/skorch#412]
Having a problem over at https://github.com/conda-forge/skorch-feedstock/pull/2
```
Processing $SRC_DIR
  Created temporary directory: /tmp/pip-req-build-op7959fv
  Added file://$SRC_DIR to build tracker '/tmp/pip-req-tracker-3oildf45'
  Running setup.py (path:/tmp/pip-req-build-op7959fv/setup.py) egg_info for package from file://$SRC_DIR
    Running command python setup.py egg_info
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-op7959fv/setup.py", line 9, in <module>
        with open('requirements.txt') as f:
    FileNotFoundError: [Errno 2] No such file or directory: 'requirements.txt'
```

...and indeed the requirements.txt doesn't seem to be in the pypi tar.gz meaning the `setup.py` can't be run from the sdist.

I *think* `requirements.txt` needs to be in the `MANIFEST.in`?

**Comment by dhirschfeld** [comment]
***xref:*** https://github.com/dnouri/skorch/pull/408

**Comment by benjamin-work** [comment]
That sounds like the problem, do you want to create a PR?

**Comment by dhirschfeld** [comment]
I'm on holiday until the new year so will be unlikely to find the time before then. Happy to do so if no one has beaten me to it though...

**Comment by BenjaminBossan** [comment]
No problem, I'll do this.

**Comment by ottonemo** [comment]
This was fixed in #413. We just released `0.5.0.post0`.

**Comment by RoyiAvital** [comment]
Is there a chance to update the `conda-forge` package?
It seems to be stuck on version [`0.11.0`](https://anaconda.org/conda-forge/skorch/files).

**Comment by BenjaminBossan** [comment]
> Is there a chance to update the `conda-forge` package?

@dhirschfeld Would you mind merging the two open PRs for skorch 0.12.0 aund 0.12.1? You could also give me permissions to the repo if you want me to do it.

**Comment by dhirschfeld** [comment]
I published the 0.12.1 version, but in general I'm stretched too thin to keep up with dependencies not on my own critical path.

As such, your help would be much appreciated @BenjaminBossan! If you'd like to be added as a maintainer just open an issue on the feedstock as described below:
***xref:*** https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-add-user-username

**Comment by BenjaminBossan** [comment]
@dhirschfeld Great, thx for the update and info. I created an issue to have myself added, no PR being created so far, let's see.

@RoyiAvital thx for notifying us, you can now install skorch 0.12.1 from conda forge.
